h/static/scripts/widget-controller.js

```diff
@@ -204,9 +242,15 @@ module.exports = function WidgetController(
     searchClients.push(searchClient);
     searchClient.on('results', function (results) {
       if (annotationUI.hasSelectedAnnotations()) {
-        // Select appropriate tab - notes or annotations, for selection
-        annotationUI.selectTab(
-          tabTypeFromSelection(annotationUI.getState().selectedAnnotationMap, results));
+        // Select appropriate tab - notes, annotations or orphans, for selection
+        $scope.$on(events.ANNOTATIONS_SYNCED, function () {
```

Comment:
I'm afraid I still don't follow why there are _two_ handlers for `events.ANNOTATIONS_SYNCED` in this module. When this event happens, both handlers will be fired and it isn't obvious when reading the code in which order those events will be fired and consequently which one will "win".